network-api/networkapi/wagtailpages/pagemodels/campaigns.py

```diff
@@ -87,6 +105,10 @@ class Callpower(TranslatableMixin, CTA):
         TranslatableField('call_button_label'),
         TranslatableField('success_heading'),
         TranslatableField('success_text'),
+        # Shareprogress fields
+        SynchronizedField('share_twitter'),
+        SynchronizedField('share_facebook'),
+        SynchronizedField('share_email'),
```

Comment:
@TheoChevalier should these be TranslatableField, since each shareprogress id is for one block of specific content?

Reply:
They should all be kept as Synchronized, as in we want to be able to override this in the admin without exposing the value to Pontoon/.po files at all. Translators — volunteers or from agencies — can’t deal with this and guess the right shareprogress ID to use here